content/docs/for-developers/tracking-events/event.md

```diff
@@ -15,10 +15,6 @@ navigation:
 
 Events are generated when email is processed by SendGrid and email service providers. There are 2 types of events - delivery and engagement events. Delivery events indicate the status of email delivery to the recipient. Engagement events indicate how the recipient is interacting with the email.
 
-Here is a flow of email events:
```

Comment:
So this is not actually unnecessary wording, but rather a broken link. Can you fix the broken link instead? Thanks!